_posts/2020-01-07-jump_dev_2020.md

```diff
@@ -64,6 +64,13 @@ Note that this code will only work until the 11th of May.
 
 [Yurii Nesterov](https://en.wikipedia.org/wiki/Yurii_Nesterov) will give a plenary during the workshop, more details on the topic and schedule will be communicated shortly.
 
+## Code of Conduct
+
+All participants will uphold the [JuMP Code of Conduct](https://github.com/JuliaOpt/JuMP.jl/blob/master/CODE_OF_CONDUCT.md)
+and the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct).
+
+To report a violation of either code of conduct, contact the program commitee at [jump-dev-2020@googlegroups.com](jump-dev-2020@googlegroups.com), or contact the [Julia Community Stewards](https://julialang.org/community/stewards/).
```

Comment:
The NumFOCUS CoC has its own contact info listed; they're not the Julia community stewards. I think we should go with the NumFOCUS contacts for this event.